executor/tasks/task_processor.py

```
@@ -68,34 +61,41 @@ def execute_task(agent: Agent) -> Tuple[TaskStatus, Optional[str]]:
+    Create an emitter for the task from execution request or task data.
 
     Supports ExecutionRequest, OpenAI format dict, and legacy dict.
 
     Args:
         request: ExecutionRequest object or task data dict
 
     Returns:
-        dict: Common callback parameters
+        ResponsesAPIEmitter: Configured emitter for the task
     """
+    from executor.config import config
```

Ruff (0.15.0)

[error] Redefinition of unused `config` from line 13: `config` redefined here

Remove definition: `config`

(F811)